s-sql/tests/tests.lisp

```diff
@@ -1441,3 +1522,64 @@ that the table will need to be scanned twice. Everything is a trade-off."
                2))
     (execute (:drop-table 'from-test-data1 :cascade))
     (execute (:drop-table 'iceland-cities :cascade))))
+
+#+abcl (test reserved-column-names-s-sql
+  (with-test-connection
```

Comment:
Wrong indentation. I suggest to put the reader conditional on a separate line, since we are conditionalizing a toplevel form.

Reply:
Agreed.